Code/Framework/AzToolsFramework/Tests/Input/QtEventToAzInputMapperTests.cpp

```diff
@@ -44,6 +44,10 @@ namespace UnitTest
             QObject::connect(m_inputChannelMapper.get(), &AzToolsFramework::QtEventToAzInputMapper::InputChannelUpdated, m_rootWidget.get(),
                 [this]([[maybe_unused]] const AzFramework::InputChannel* inputChannel, QEvent* event)
                 {
+                    if(event == nullptr) {
+                        return;
+                    }
```

Comment:
standards: Braces always go on a new line

```
if(event == nullptr)
{
    return;
}
```
(more info here - https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md)